include/conversion.h

```diff
@@ -0,0 +1,9 @@
+#ifndef CONVERISON_H
+#define CONVERISON_H
+
+#include "common.h"
+
+size_t bin_to_agbr(char** bufp, size_t size);
```

Comment:
small typo, this should be `abgr`

Reply:
If this typo is consistent throughout, please correct it everywhere and let me know. Sorry for the significant delay on reviewing this!